contracts/IdentityFactory.sol

```diff
@@ -11,7 +12,7 @@ contract IdentityFactory {
     mapping(address => address) public senderToProxy;
 
     //cost ~2.4M gas
-    function CreateProxyWithControllerAndRecovery(address userKey, address[] delegates, uint longTimeLock, uint shortTimeLock) {
+    function createProxyWithControllerAndRecovery(address userKey, address[] delegates, uint longTimeLock, uint shortTimeLock) {
```

Comment:
Do we have to update anything in mobile app b/c of these changes? Just wondering.